policyengine_us/parameters/gov/irs/gross_income/pre_tax_contributions.yaml

```
@@ -0,0 +1,9 @@
+description: The US subtracts these payroll deductions from wages and salaries when determining the taxable amount.
+values:
+  2010-01-01:
+    # Assumes all are pre-tax.
+    - retirement_contributions
```

```suggestion
    - retirement_contributions
```
`retirement_contributions` does not exists yet, are we looking for `pension_contributions` which is the sum of `ira_contributions` and `sep_simple_qualified_plan_contributions`?

---

We'll split this in cps.py